src/composables/multisigAccountCreate.ts

```diff
@@ -135,6 +138,9 @@ export function useMultisigAccountCreate({ store }: IDefaultComposableOptions) {
     const creationFeeUnformatted = new BigNumber(outerFee).plus(innerFee).toFixed();
     multisigAccountCreationFee.value = Number(AmountFormatter.toAe(creationFeeUnformatted));
     multisigAccountCreationPhase.value = MULTISIG_CREATION_PHASES.signed;
+    notEnoughBalanceToCreateMultisig.value = (
+      balances.value[payerId]?.isLessThan(multisigAccountCreationFee.value)
+    );
```

Comment:
This calculation could be done as `computed` in the respective view components.

Reply:
Because in the new update, the payer account is isolated from the current active account and we can't access the payerId in `[MultisigVaultCreate.vue](https://github.com/aeternity/superhero-wallet/pull/1888/files#diff-6342e78347a20aa189ed26026bf0d0cb81995b9cf436ac0fe4a50b47799449a1)` had to perform this calculation here.  also, we are doing the fee calculation in the function.

Reply:
I see. sounds good!